eng/common/scripts/Collect-ChangeLogs.ps1

```diff
@@ -0,0 +1,60 @@
+[CmdletBinding()]
+param(
+  [Parameter(Mandatory=$true)]
+  [String] $FromDate # In the format "yyyy-MM-dd"
+)
+
+. (Join-Path $PSScriptRoot common.ps1)
+
+$releaseHighlights = @{}
+
+if ($FromDate -as [DateTime])
+{
+    $date = ([DateTime]$FromDate).ToString($CHANGELOG_DATE_FORMAT)
+}
+else {
+    LogWarning "Invalid date passed. Switch to using the current date"
```

Comment:
I think that there is a difference between a typo, resulting in the wrong date (current) being used, a valid date and no date being used. Better to be explicit on your inputs ... you either provide a valid date, or no date ... an invalid date should be an error because it might be accidental.